content/3_Silver/2P.mdx

```diff
@@ -31,6 +31,144 @@ frequency: 2
 	</Resource>
 </Resources>
 
+## Two Pointers
+
+The two pointers method iterates two pointers across an array, to track the start and end
+of an interval, or two values in a sorted array that we are currently checking. 
+Typically, each pointer starts at seperate ends and moves strictly in one direction. 
+However, as shown in the focus problem below, the two pointers do not have to start at seperate ends.
+
+
+<FocusProblem problem="book" />
+
+## Solution - Books
+
+We want to find the longest continuous segment of books that can be read within $t$ minutes. 
+
+To accomplish this, we can define $\texttt{left}$ and $\texttt{right}$ as the start and end of the segment respectively.
+
+Since being able to read segment $\texttt{left} \dots \texttt{right}$ implies we can read segments $\texttt{left} \dots \texttt{right}-1$, we can use two-pointer (or, alternatively, [binary search](/silver/binary-search)).
```

Comment:
```suggestion
Since being able to read segment $\texttt{left} \dots \texttt{right}$ implies we can read segment $\texttt{left} \dots \texttt{right}-1$, we can use two-pointers (or, alternatively, [binary search](/silver/binary-search)).
```